packages/terra-date-picker/src/_PopperContainer.scss

```diff
@@ -0,0 +1,7 @@
+:local {
+  .popper-container {
+    position: absolute;
+    top: 0;
+    z-index: 2147483647;
```

Comment:
Is the intent to set this to a value so high, z-index values on other elements wouldn't be higher?

Reply:
Yeah. It's the same z-index that tether originally set so that it's widgets would display on top.

Reply:
> It's the same z-index

We should add an inline comment mentioning that